tests/Drupal/bad/BadUnitTest.php

```diff
@@ -384,7 +384,7 @@ protected function getErrorList(string $testFile): array
                 849 => 2,
                 860 => 2,
                 867 => 1,
-                871 => 2,
+                879 => 2,
```

Comment:
Since this sniff will likely not interfere with others we should leave BadUnitTest alone and do this in a dedicated test file for this new sniff.